backend/src/main/java/com/crm/exception/user/NoSuchAccountException.java

```diff
@@ -0,0 +1,8 @@
+package com.crm.exception.user;
+
+public class NoSuchAccountException extends CrmLoginException {
```

Comment:
Shouldn't it return a status code 404 instead of 400?

Reply:
I think you right.